4.6/Dockerfile

```diff
@@ -60,10 +48,14 @@ RUN set -x \
 	&& sed -ri "s!^(\#\s*)?(elasticsearch\.url:).*!\2 'http://elasticsearch:9200'!" /opt/kibana/config/kibana.yml \
 	&& grep -q 'elasticsearch:9200' /opt/kibana/config/kibana.yml
 
+RUN chmod 666 /opt/kibana/optimize/.babelcache.json
+RUN chmod 777 /opt/kibana/config
```

Comment:
These need to be in the layer that creates the files, otherwise graph drivers like `aufs` will fail since the file can't be more permissive in a higher layer.

Don't all the versions need these chmods?